packages/react/src/components/errors/Error.styles.ts

```diff
@@ -0,0 +1,35 @@
+const FONT_SANS_SERIF = `-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Helvetica, Arial, system-ui, sans-serif`
```

Comment:
I stole these font choices from the uncaught error overlay in the `sanity` repo. I'm not sure if this is the best place for them but I put them here for now since it's the only styles file.